### DISCUSSION TITLE
AI takeover in education: sitting in on Pandora’s classroom [AI2728P]

### DISCUSSION THREAD

**Discussion by kvignali** [community/community#74412] [GitHub Education]
## AI takeover in education: sitting in on Pandora’s classroom

**Moderator:** Kaitlin Vignali, GitHub Product Lead, Education
 
**Panelists:** 
- Joel Hammer, Chair of Computer Science at the Friends School of Baltimore
- Samyuktha Gopalsamy, Masters in Computer Science from Rutgers, software engineer at Moody’s Analytics
- Brook Riggio, VP of Education and Cofounder at Code Fellows
- Ignacio Palma, student leader of the The Artificial Intelligence Club @ Pontificia Universidad Católica de Chile and GitHub Campus Expert

> Artificial Intelligence (AI) has been on a transformative rollercoaster over the past two years. While it has been used to personalize learning experiences and increase student proficiency, some worry that unchecked AI has the potential to unravel fundamentals and introduce bias within the classroom. In this panel discussion, we will explore various sides of AI in education from both student and teacher perspectives, including how AI is being used to improve student outcomes and create more equitable learning environments. We will also explore the ethical implications of using AI in education and how we can ensure that it is used responsibly.

 Thank you for joining our panel! If you have any questions or feedback regarding, please feel free to add in the comments and we'll do our best to answer.

**Comment by crguezl** [comment]
Thanks for sharing your thoughts!

The youtube video is [here](https://youtu.be/ekaxwMqdywA?si=CkTyhJWMzdoFezxe)
